homeassistant/components/webostv/notify.py

```diff
@@ -55,6 +49,7 @@ async def async_send_message(self, message="", **kwargs):
         except (
             OSError,
             ConnectionClosed,
+            ConnectionRefusedError,
             asyncio.TimeoutError,
             asyncio.CancelledError,
             PyLGTVCmdException,
```

Comment:
Looks like this is set of exceptions are used in another place.  Would be good to make them a constant